src/install/dracut-install.c

```diff
@@ -398,7 +398,7 @@ static int library_install(const char *src, const char *lib)
 
         strcpy(p, lib);
 
-        clib = strjoin(ppdir, "/", basename(p), NULL);
+        clib = strjoin(ppdir, "/", basename(p), (const char *)NULL);
```

Comment:
This is pointless. NULL is automatically converted to a pointer to any type.

Reply:
Yes but C standards doesn't strictly mandate NULL to be (void *)0, it can be simply 0.
https://stackoverflow.com/questions/35107035/variable-arguments-list-and-null-pointer
I think that's the reason cppcheck threw this warning

Reply:
The C standard may not, but POSIX does. I suppose that's good enough for dracut.

Reply:
So, you want me to undo this? or let it be?

Reply:
Undo. Why change something that's not really a problem?

Reply:
Cool, ty.